pkgs/top-level/all-packages.nix

```diff
@@ -1489,6 +1489,8 @@ with pkgs;
     utillinux = utillinuxMinimal;
   };
 
+  sqlint = callPackage ../development/tools/sqlint { };
```

Comment:
Not sure how these are intended to be sorted. Why do we go back to `a` below here?

Reply:
Sometimes packages of similar category are kept together. Sometimes they get out of order due merges being delayed.
We still try to keep them alphabetical since this avoids merge conflicts.